test/fork/recalculate-effective-weights-bug-fix.js

```diff
@@ -0,0 +1,128 @@
+const { ethers } = require('hardhat');
```

Comment:
i made a separate file from @kyledewy  to just test this particular upgrade but reused his verification logic.

Happy to adjust if you think there should just be 1.